tests/e2e/script_test.go

```diff
@@ -139,7 +147,38 @@ func TestE2EScript(t *testing.T) {
 							h.T.Logf("skipping MockGCPBackdoor command, because not running against mockgcp")
 						}
 
-						captureHTTPLogEvents()
+						captureHTTPLogEvents(false)
+						continue
+					}
+
+					// SystemRun let's KCC run for a while to observe/ gather HTTP logs
+					if obj.GroupVersionKind().Kind == "SystemRun" {
+						if h.MockGCP != nil {
+							d, _, _ := unstructured.NestedInt64(obj.Object, "duration")
+							waitTimeout := time.Duration(d) * time.Second
+							timeoutChan := time.After(waitTimeout)
+							ticker := time.NewTicker(30 * time.Second)
+
+							for {
+								stopWaiting := false
+								select {
+								case <-timeoutChan:
+									t.Logf("finished waiting for http log collections")
+									stopWaiting = true
+									break
+								case <-ticker.C:
+									t.Logf("waiting for http log collections")
+								}
+								if stopWaiting {
+									break
+								}
+							}
+
+						} else {
+							h.T.Logf("skipping MockGCPBackdoor command, because not running against mockgcp")
+						}
+
+						captureHTTPLogEvents(true)
```

Comment:
What are we expecting to see here?  I would think we would normally want to check it (i.e. not skip), because everything should be deterministic?